### DISCUSSION TITLE
Aderyn AI for security research

### DISCUSSION THREAD

**Discussion by TilakMaddy** [Cyfrin/security-and-auditing-full-course-s23#29] [Show and tell]
Hey everyone,
First of all, thank you for reading!

This is if you are familiar with [Aderyn](https://github.com/Cyfrin/aderyn) and you are looking for something that can accelerate your pace I implore you to check out [Aderyn AI](https://github.com/TilakMaddy/aderyn-ai) which is designed to act like your personal assistant when looking at the report document. 

Thank you.

**Comment by drained69** [comment]
Thanks for sharing.

**Reply by TilakMaddy** [reply]
You are welcome ! Glad to help

**Reply by cromewar** [reply]
@This is Sick! @TilakMaddy
